src/main/java/org/elasticsearch/indices/recovery/RecoveryState.java

```diff
@@ -811,15 +812,7 @@ public synchronized long totalRecoverBytes() {
             return total;
         }
 
-        public synchronized long totalReuseBytes() {
```

Comment:
the fact that this is not reused is actually a bug in the toXContent:

```
builder.byteSizeField(Fields.TOTAL_IN_BYTES, Fields.TOTAL, totalBytes());
 builder.byteSizeField(Fields.REUSED_IN_BYTES, Fields.REUSED, totalBytes());
            
```

can we keep this here? I'll make a pull request..